week-4/Homework/mandatory/2-writers.js

```diff
@@ -39,6 +39,10 @@ let writers = [
   }
 ];
 
+writers.filter(writer => writer.alive).forEach(writer => console.log(
+ `Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.`));
+

```

Comment:
this code does only console out the writers who are alive in the formatted sentence,  but the homework question is asking you to do a different task:
1-Loop through the array, 
and for each object, `console.log()` out the sentence:

"Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
2- and the extra challenge is to console out the writers that are alive.